full-stack-tests/backend/src/integration/AzuriteTest.ts

```diff
@@ -4,13 +4,14 @@
 *--------------------------------------------------------------------------------------------*/
 
 import { expect } from "chai";
+import * as crypto from "crypto";
```

Comment:
For imports from node, you can use `from "node:crypto"` or `from "node:fs"` to add clarity as to where these utils are coming from. It is functionally the same, but makes it easier to read.

Reply:
resolved in [commit 7ad75934bda38b7d23a0730443088800a94e3716](https://github.com/iTwin/itwinjs-core/pull/7000/commits/7ad75934bda38b7d23a0730443088800a94e3716)